stream/chunk_writer.js

```diff
@@ -48,7 +57,10 @@ ChunkWriter.prototype._transform = function _transform(value, encoding, callback
     var err = tup[0];
     var buffer = tup[1];
     if (err) {
-        callback(err);
+        callback(ChunkWriteError(err, {
+            valueInspected: inspect(value),
```

Comment:
You might consider {depth} options to inspect. Defaults are somewhat shallow, though this can be a boon if the object inspected has a lot of breadth.